chaosk8s/actions.py

```diff
@@ -121,3 +122,33 @@ def scale_microservice(name: str, replicas: int, ns: str = "default",
         raise ActivityFailed(
             "failed to scale '{s}' to {r} replicas: {e}".format(
                 s=name, r=replicas, e=str(e)))
+
+
+def kill_statefulset(name: str, ns: str = "default",
```

Comment:
Why `kill_` ?

Reply:
This should live under `chaosk8s/staetfulset/actions.py`

Then add a line to:
* https://github.com/chaostoolkit/chaostoolkit-kubernetes/blob/master/setup.py#L48
* https://github.com/chaostoolkit/chaostoolkit-kubernetes/blob/master/chaosk8s/__init__.py#L131

Reply:
> Why `kill_` ?

Do you prefer 'delete_statefulset' (like kubernetes-client function delete_namespaced_stateful_set) or 'remove_statefulset' (like chaostoolkit-kubernetes function remove_service_endpoint) ?

Reply:
I use remove_statefulset to stay coherent with chaostoolkit-kubernetes